.github/workflows/build_site.yml

```
@@ -49,6 +49,15 @@ jobs:
           bundle install
       - name: Build website
         run: bundle exec jekyll build
+      - name: Build the JupyterLite site
+        run: |
+          rm -rf `find . -type d -name .pytest_cache`
+          rm -rf `find . -type d -name .ruff_cache`
+          rm -rf `find . -type d -name .doctrees`
+          mkdir content
+          mkdir _site/jupyter-lite
+          cp -R ch*/*.ipynb content
```

Maybe we should keep the directories, to make it easier to browse all the files. Maybe as
```suggestion
          find ./ -wholename 'ch*/*.ipynb' -exec cp --parents {}  content \;
```